### PR TITLE
fix(executor): Fix emissary bugs

### PR DESCRIPTION
**Pull request by alexec**

### CODE REVIEW

File: workflow/executor/os-specific/signal_windows.go [hunk 1]
```diff
@@ -8,3 +8,22 @@ import (
...
 	return syscall.SIGINT
 }
+
+func IsSIGCHLD(s os.Signal) bool {
+	return false // this does not exist on windows
+}
+
+func Kill(pid int, s syscall.Signal) error {
+	if pid < 0 {
+		pid = -pid // // we cannot kill a negative process on windows
```
Comment (lippertmarkus) [inline workflow/executor/os-specific/signal_windows.go:18]: duplicated `//`

**Review by lippertmarkus — APPROVED** [review]
should work

**Comment by alexec** [comment]
@whynowy can I please get approval?

**Comment by alexec** [comment]
@lippertmarkus FYI - I don't expect the Emissary to work on Windows because I' not able to test it. Would you or anyone else  @argoproj/argo-windows like to try it out?

**Review by whynowy — APPROVED** [review]
LGTM

**Pull request merged**